corehq/apps/hqwebapp/static/hqwebapp/js/knockout_subscribables.ko.js

```diff
@@ -14,3 +14,16 @@ ko.subscribable.fn.snakeCase = function(re) {
         owner: this,
     }).extend({ notify: 'always' });
 };
+
```

Comment:
Any chance you could change this to an hqDefine module? Looks like it'll be trivial.